tasks/main.yml

```diff
@@ -32,6 +32,26 @@
   changed_when: False
   failed_when: False
 
+- name: singularity | remove older Go installation
+  file:
+    path: '/opt/go/{{ item }}'
+    state: absent
+  with_items: "{{ old_go_versions }}"
+  
+- name: singularity | install dependencies
+  apt:
+    pkg:
+    - build-essential
+    - uuid-dev
+    - libgpgme-dev
+    - squashfs-tools
+    - libseccomp-dev
+    - wget
+    - pkg-config
+    - git
+    - cryptsetup-bin
+    state: latest
```

Comment:
Pleas look here:  https://github.com/abims-sbr/ansible-singularity/blob/master/vars/Debian.yml

Reply:
Ok, I'll update the list

Reply:
There is also on for redhat. You need to find the equivalents